content/en/docs/setup/best-practices/node-conformance.md

```diff
@@ -25,10 +25,10 @@ daemons installed:
 ## Running Node Conformance Test
 
 To run the node conformance test, perform the following steps:
-
-1. Point your Kubelet to localhost `--api-servers="http://localhost:8080"`,
-because the test framework starts a local master to test Kubelet. There are some
-other Kubelet flags you may care:
+1. Get the value of `--kubeconfig` flag from your Kubelet, for example:
+`--kubeconfig=/var/lib/kubelet/config.yaml`,use `http://localhost:8080` as the
+API-Server address.Because the test framework starts a local master to test Kubelet.
+There are some other Kubelet flags you may care:
```

Comment:
How about:
```suggestion
1. Work out the value of the `--kubeconfig` option for the kubelet; for example:
   `--kubeconfig=/var/lib/kubelet/config.yaml`.
    Because the test framework starts a local control plane to test the kubelet,
    use `http://localhost:8080` as the URL of the API server.
    There are some other kubelet command line parameters you may want to use:
```
?

Reply:
The `--api-servers` flag has been moved to `--kubeconfig` for configuration.

According to the current document, the test server is running locally and the tag needs to be changed.

Reply:
@Cweiping , What do you think about updating the changes with the suggested change?
The suggestion corrects the formatting and fixes a few style issues. Thanks!

Reply:
I don't understand. Can you make it clearer?
@kbhawkey